ibm_was/README.md

```diff
@@ -0,0 +1,53 @@
+# Agent Check: IBM WAS
+
+## Overview
+
+This check monitors [IBM WAS][1].
+
+## Setup
+
+The IBM WAS Datadog Integration collects enabled PMI Counters from the WebSphere Application Server environment. Setup requires enabling the PerfServlet as documented on the [IBM documentation][8]
+
+### Installation
+
+The Ibm_was check is included in the [Datadog Agent][2] package, so you do not
+need to install anything else on your server.
+
+### Configuration
+
+1. Edit the `ibm_was.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your ibm_was performance data.
+   See the [sample ibm_was.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3]
+
+### Validation
+
+[Run the Agent's `status` subcommand][4] and look for `ibm_was` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
+
+### Service Checks
+
+See [service_checks.json][7] for a list of service checks provided by this check
```

Comment:
I tested this with the docs site build, it doesn't work. I will submit a fix commit.

Reply:
Thanks, is the docs side all good here?